### DISCUSSION TITLE
Location Access All The Time

### DISCUSSION THREAD

**Discussion by GitSnider** [owntracks/android#1344] [Q&A]
Is there a possibility of getting the app updated for Android to have the option to allow location access all the time? I’ve seen that option on other apps but it seems to only have while using or deny as choices on this app. I use the app to register presence in my Hubitat landscape. That’s why I’d like the all of the time option.

Regards,
Stu

**Comment by growse** [comment]
Does OT not send location updates when background currently? It should do (and does for me and many others).

What device do you have?
